components/BookCover.vue

```diff
@@ -135,6 +135,9 @@ const coverClass = computed(() => {
   else {
     classes.push('pointer-events-none')
   }
+  if (hasError.value) {
+    classes.push('h-full')
```

Comment:
but this is `absolute` and `inset-0`, how does it help?

Reply:
<img width="313" alt="image" src="https://github.com/user-attachments/assets/c1265c61-e27b-480d-86a3-1f5efdc520f4" />
without h-full